README.md

```diff
@@ -20,7 +20,7 @@ A fluid multi-table layout. Each table groups subjects together under a shared p
 
 1. Copy this lens into your `refocus-ldk/Lenses` directory.
 
-        cp -r ../refocus-lens-multitable/MultiTable Lenses/MultiTable/
+        cp -r ../refocus-lens-multitable/MultiTable Lenses/
```

Comment:
I'm not sure about this change... doesn't it depend on what directory you're in when you run this command?

Reply:
I don't think so. I tried this from refocus-ldk directory (which is what we are instructing in README, right?), and it led to creation of a directory "MultiTable" inside Lenses/MultiTable. That is why my lens was not getting updated with changes.

I think it should be either of these:
cp -r ../refocus-lens-multitable/MultiTable/ Lenses/MultiTable/   (notice added / after ../refocus-lens-multitable/MultiTable)
or
cp -r ../refocus-lens-multitable/MultiTable Lenses/